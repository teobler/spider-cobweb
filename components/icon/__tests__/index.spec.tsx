import React from 'react';
import renderer from 'react-test-renderer';
import Icon from '../index';

describe('icon', () => {
  it('should render a empty icon', () => {
    const emptyIconJson = renderer.create(<Icon name="wechat"/>).toJSON();
    expect(emptyIconJson).toMatchSnapshot();
  });
});
