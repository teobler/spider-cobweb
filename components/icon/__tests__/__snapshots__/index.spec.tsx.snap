// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon should render a empty icon 1`] = `
<svg
  className="spider-icon"
>
  <use
    xlinkHref="#wechat"
  />
</svg>
`;
